docs/content/learn/tutorials/general/extension.md

```diff
@@ -0,0 +1,770 @@
+{{% /tab %}}
+{{% tab %}}
+{{< highlight json "linenos=table" >}}
+{
+  "props": [
+    {
+      "name":"...",
+      "ns":"...",
+      "value":"...",
+      "class":"..."
+    }
+  ]
+}
+{{< /highlight >}} 
+
+A listing of properties in OSCAL JSON are represented using the `props` object array, with each property defined as an individual JSON object array member.
+
+Each *property* object can have the following JSON property names.
+
+- `name` (Required) - The required *name* of the OSCAL property that must be a [token](/reference/datatypes/#token) data type.
+- `ns` (Optional) - The optional *namespace* of the OSCAL property that must be a [universal resource identifier (URI)](/reference/datatypes/#uri) formatted according to [RFC 3986](https://datatracker.ietf.org/doc/html/rfc3986).
+- `value` (Required) - The required *value* of the OSCAL property that must be a [string](/reference/datatypes/#string) value.
+- `class` (Optional) - The optional *class* of the OSCAL property  that must be a [string](/reference/datatypes/#string) value, which can be used to further qualify the value that must be a [string](/reference/datatypes/#string) value, which can be used to further qualify the property's value.
+{{% /tab %}}
+{{% tab %}}
+{{< highlight yaml "linenos=table" >}}
+props:
+- name: ...
+  ns: ...
+  value: ...
+  class: ...
+{{< /highlight >}} 
+
+A `props` list in YAML contains individual OSCAL property items.
+
+Each property item can have the following keys:
+
+- `name` (Required) - The required *name* of the OSCAL property that must be a [token](/reference/datatypes/#token) data type.
+- `ns` (Optional) - The optional *namespace* of the OSCAL property that must be a [universal resource identifier (URI)](/reference/datatypes/#uri) formatted according to [RFC 3986](https://datatracker.ietf.org/doc/html/rfc3986).
+- `value` (Required) - The required *value* of the OSCAL property that must be a [string](/reference/datatypes/#string) value.
+- `class` (Optional) - The optional *class* of the OSCAL property  that must be a [string](/reference/datatypes/#string) value, which can be used to further qualify the value that must be a [string](/reference/datatypes/#string) value, which can be used to further qualify the property's value.
+{{% /tab %}}
+{{% /tabs %}}
+
+### Concepts
+
+Now that the syntax of a property is more clear, let's discuss how the property's *namespace*, *name*, *value*, and *class* are used to extend OSCAL.
+
+#### Context
+
+As we mentioned earlier, a property is a *namespaced name* and *value* pair, which appears on many of the key fields within an OSCAL model. The OSCAL field on which the property is declared is considered the property's *context*. A property's context establishes the subject of the property; the thing the property is about.
```

Comment:
Punctuation should be a colon not semi-colon

> the subject of the property: the thing the property is about.

Reply:
```suggestion
As we mentioned earlier, a property is a *namespaced name* and *value* pair, which appears on many of the key fields within an OSCAL model. The OSCAL field on which the property is declared is considered the property's *context*. A property's context establishes the subject of the property: the thing the property is about.
```